draft-bormann-jsonpath-iregexp.md

```diff
@@ -93,6 +96,9 @@ This document includes rules for converting I-Regexps for use with several well-
 
 {::boilerplate bcp14-tagged}
 
+The grammatical rules in this document are to be interpreted as ABNF,
+as described in {{-abnf}}.
```

Comment:
Actually, the current ABNF needs RFC 7405 as well.

Reply:
Heh, I had no idea 7405 even existed.  OK.

Reply:
Having read 7405, it's a delta against 5234 so strictly I guess only 7405 need be referenced but it seems more polite to reference both.

Reply:
Indeed, that's what I usually do.